### DISCUSSION TITLE
Null modified date in recent commit

### DISCUSSION THREAD

**Discussion by isstabb** [SigmaHQ/sigma#2866] [General]
https://github.com/SigmaHQ/sigma/blob/adce5eff93caf002d7df1b098f9c42c68c41b244/rules/windows/process_creation/proc_creation_win_susp_powershell_iex_patterns.yml#L7

https://github.com/SigmaHQ/sigma/commit/cc457436694499881355f22bd1d3964f60cbcd99

It threw off our automation, otherwise I guess no big deal? @Neo23x0

**Comment by nasbench** [comment]
Fixed
